src/webapp/pages/landing/useLandingPage.ts

```diff
@@ -19,7 +19,7 @@ export function useLandingPage() {
 
     const [isSettingsOpen, { enable: openSettings, disable: closeSettings }] =
         useBooleanState(false);
-    const [includeHeaders, setIncludeHeaders] = React.useState(false);
+    const [includeHeaders, setIncludeHeaders] = React.useState(true);
```

Comment:
Not wrong, but just as a note, we have useBooleanState

Reply:
Forgot again 😅🙏🏼